---a/net/minecraft/block/BlockHalfStoneSlabNew.java
+++b/net/minecraft/block/BlockHalfStoneSlabNew.java
@@ -1,12 +1,11 @@
 package net.minecraft.block;
 
-public class BlockHalfStoneSlabNew extends BlockStoneSlabNew
-{
+public class BlockHalfStoneSlabNew extends BlockStoneSlabNew {
+
     private static final String __OBFID = "CL_00002110";
 
-    public boolean isDouble()
-    {
+    public boolean isDouble() {
         return false;
     }
-}
+}
